app/search.py

```diff
@@ -0,0 +1,54 @@
+import os
+import requests
+from dotenv import load_dotenv
+from flask import current_app as app
+
+load_dotenv()
```

Comment:
Why are we doing this here? Might be better to centralize setup like this so its easy to tell at a glance what happens as the app starts up